### DISCUSSION TITLE
Sorry to be a bother but could you maybe make a Mac OS high sierra .utm file?

### DISCUSSION THREAD

**Discussion by Sahelalemi** [utmapp/UTM#5852] [General]
Well, I’m not sure about others but I’d love to have a high sierra .utm file, and since your the only person who actually formatted theses into .utm files, I came and asked you. I’m so sorry if I’m annoying you.

sahel

**Comment by F1248** [comment]
[Here](https://github.com/adespoton/utmconfigs) are configurations from Mac OS 9 to macOS Monterey 12, including [macOS High Sierra](https://github.com/adespoton/utmconfigs/blob/main/macOS%2010.13%20(config%20only).utm.zip).

**Comment by Sahelalemi** [comment]
Is here one where you wouldn’t need an iso?

**Comment by F1248** [comment]
> Is here one where you wouldn’t need an iso?

You always need an ISO! You can create one with [MIST](https://github.com/ninxsoft/Mist), but only on an Intel Mac.

**Comment by Sahelalemi** [comment]
Oh it’s cuz when I try importing .iso files it doesn’t work, even when I click import iso file, it only lets me access .utm?

**Comment by F1248** [comment]
> Oh it’s cuz when I try importing .iso files it doesn’t work, even when I click import iso file, it only lets me access .utm?

You must first import the .utm file and then add the .iso file in the settings of this VM.

**Comment by Sahelalemi** [comment]
Okay
